halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/artifacts/s3/S3ArtifactAccount.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Datadog, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.config.model.v1.artifacts.s3;
+
+import com.netflix.spinnaker.halyard.config.model.v1.node.ArtifactAccount;
+
+public class S3ArtifactAccount extends ArtifactAccount {
+    public String name;
+
+    public String getName() {
```

Comment:
Sorry, one last thing I missed. I would drop this getter, and annotate the class with `@Data`, which will generate everything this class needs to be correctly serialized & deserialized by Halyard